### DISCUSSION TITLE
asset manager cpp

### DISCUSSION THREAD

**Discussion by antn9x** [axmolengine/axmol#1287] [General]
is there any tutorial or documents about asset manager in cpp? i don't know how to use it, any sample code?

**Comment by rh101** [comment]
Check this sample project: https://github.com/syuhari/AssetsManagerEx

That sample uses Cocos2d-x, but it will give you an idea on how to use `AssetManagerEx` with Axmol.

**Reply by antn9x** [reply]
very clear example, i'll try this code, thank you.

**Reply by aismann** [reply]
@antn9x 
Is this the answer? plz mark it.
